src/pybind/mgr/dashboard_v2/tests/test_tools.py

```diff
@@ -30,6 +30,14 @@ def delete(self, key):
     def bulk_delete(self):
         FooResource.elems = []
 
+    def set(self, data, key):
+        FooResource.elems[int(key)] = data
+        return dict(key=key, **data)
+
+    @detail_route(methods=['get'])
```

Comment:
why not use the cherrypy annotations like:

```
@cherrypy.expose
def detail(self, key):
    # ....
```

Reply:
I don't know! Is it possible to expose `/api/cluster/<fsid>/status`, but not  `/api/cluster/status` with `@cherrypy.expose` ?

Reply:
I don't know either